openslides/users/static/js/users/site.js

```diff
@@ -493,8 +493,9 @@ angular.module('OpenSlidesApp.users.site', [
     'osTableSort',
     'gettext',
     'UserPdfExport',
+    'ErrorMessage',
     function($scope, $state, $http, $q, ngDialog, UserForm, User, Group, PasswordGenerator, Projector, ProjectionDefault,
-        Config, gettextCatalog, UserCsvExport, osTableFilter, osTableSort, gettext, UserPdfExport) {
+        Config, gettextCatalog, UserCsvExport, osTableFilter, osTableSort, gettext, UserPdfExport, ErrorMessage) {
```

Comment:
I can't see any error message in users list view if the server is offline... Can you check it please?